packages/alpha/src/types.ts

```diff
@@ -230,6 +230,45 @@ export interface SimulationRun {
   lastUpdatedTime: Date;
 }
 
+export type SimulationRunDataValueType =
+  | 'STRING'
+  | 'DOUBLE'
+  | 'STRING_ARRAY'
+  | 'DOUBLE_ARRAY';
+
+export const SimulationRunDataValueType = {
+  STRING: 'STRING' as SimulationRunDataValueType,
+  DOUBLE: 'DOUBLE' as SimulationRunDataValueType,
+  STRING_ARRAY: 'STRING_ARRAY' as SimulationRunDataValueType,
+  DOUBLE_ARRAY: 'DOUBLE_ARRAY' as SimulationRunDataValueType,
+};
+
+export interface SimulationRunDataOutput {
+  referenceId: CogniteInternalId;
+  value: string | number | string[] | number[];
+  valueType: SimulationRunDataValueType;
+  unit: {
+    name: string;
+    externalId?: CogniteExternalId;
+  };
```

Comment:
I think the unit is optional
```suggestion
  unit?: {
    name: string;
    externalId?: CogniteExternalId;
  };
```

Reply:
Good catch, fixed that and another one.